index.html

```diff
@@ -3995,9 +3995,6 @@ <h2>Elements</h2>
   the following <a><code>ScrollIntoViewOptions</code></a>:
 
   <dl>
-   <dt>"<code>behavior</code>"
-   <dd>"<code>instant</code>"
```

Comment:
This seems to come from https://github.com/w3c/webdriver/pull/1493

Reply:
yea... I wasn't watching my branches... school boy error

Reply:
Probably shouldn't be here, then :)